doc/nrf/ug_nrf5340.rst

```diff
@@ -589,6 +591,42 @@ Additionally, the memory partitions must be defined and include:
 Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
 To learn more about Thingy:53, see :ref:`ug_thingy53`.
 
+MCUboot's serial recovery of the networking core image
+******************************************************
+
+In addition to the recovery of the application core image, also the networking core image can be recovered.
```

Comment:
Do we have the application core image recovery described anywhere? This reference to the application core image begs for a link here...

Reply:
Serial recovery is the same serial protocol as used by the mcumgr/smp_svr sample. We don't have any doc on that,

Reply:
Isn't this the protocol description? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/services/device_mgmt/smp_protocol.html